roles/redhat_satellite6_capsule_lb/defaults/main.yml

```diff
@@ -0,0 +1,11 @@
+---
+# defaults file for satellite_lb
+domain: example.com
+sat_server: sat6-master-01.{{ domain }}
+capsule1: sat6-capsule-01.{{ domain }}
+capsule2: sat6-capsule-02.{{ domain }}
+org: ORG1
+location: LOC1
+cert_path1: /tmp/{{ capsule1 }}.tar
+cert_path2: /tmp/{{ capsule2 }}.tar
+lb_server: capsule.{{ domain }}
```

Comment:
lets at the very least prefix all variables with `satellite_` due to the craziness that is variable precedence in Ansible to help scope the variables.